src/act/declaration.rs

```diff
@@ -0,0 +1,52 @@
+use proc_macro2::TokenStream;
+use std::collections::HashMap;
+
```

Comment:
ACT create_identifier should probably return None

Reply:
We decided to get rid of the Procalim trait implementation for ACT all together